simple-stripe-payments.php

```diff
@@ -41,80 +41,80 @@
 
-if ( !array_key_exists( 'test_mode', $stripe_options ) ) {
-	$stripe_options['test_mode'] = 'false';
+if ( !array_key_exists( 'test_mode', $ssp_options ) ) {
+	$ssp_options['test_mode'] = 'false';
 }
 
-if ( !array_key_exists( 'live_secret_key', $stripe_options ) ) {
-	$stripe_options['live_secret_key'] = '';
+if ( !array_key_exists( 'live_secret_key', $ssp_options ) ) {
+	$ssp_options['live_secret_key'] = '';
 }
 
-if ( !array_key_exists( 'live_publishable_key', $stripe_options ) ) {
-	$stripe_options['live_publishable_key'] = '';
+if ( !array_key_exists( 'live_publishable_key', $ssp_options ) ) {
+	$ssp_options['live_publishable_key'] = '';
 }
 
-if ( !array_key_exists( 'test_secret_key', $stripe_options ) ) {
-	$stripe_options['test_secret_key'] = '';
+if ( !array_key_exists( 'test_secret_key', $ssp_options ) ) {
+	$ssp_options['test_secret_key'] = '';
 }
 
-if ( !array_key_exists( 'test_publishable_key', $stripe_options ) ) {
-	$stripe_options['test_publishable_key'] = '';
+if ( !array_key_exists( 'test_publishable_key', $ssp_options ) ) {
+	$ssp_options['test_publishable_key'] = '';
 }
-if ( !array_key_exists( 'form_button_color', $stripe_general_settings ) ) {
-    $stripe_general_settings['form_button_color'] = '';
+if ( !array_key_exists( 'form_button_color', $ssp_general_settings ) ) {
+    $ssp_general_settings['form_button_color'] = '';
 }
 
-if ( !array_key_exists( 'form_button_hover_color', $stripe_general_settings ) ) {
-    $stripe_general_settings['form_button_hover_color'] = '';
+if ( !array_key_exists( 'form_button_hover_color', $ssp_general_settings ) ) {
+    $ssp_general_settings['form_button_hover_color'] = '';
 }
 
-if ( !array_key_exists( 'form_button_title_color', $stripe_general_settings ) ) {
-    $stripe_general_settings['form_button_title_color'] = '';
+if ( !array_key_exists( 'form_button_title_color', $ssp_general_settings ) ) {
+    $ssp_general_settings['form_button_title_color'] = '';
 }
 
-if ( !array_key_exists( 'form_button_title_hover_color', $stripe_general_settings ) ) {
-    $stripe_general_settings['form_button_title_hover_color'] = '';
+if ( !array_key_exists( 'form_button_title_hover_color', $ssp_general_settings ) ) {
+    $ssp_general_settings['form_button_title_hover_color'] = '';
 }
 
 /**********************************
 * includes
 **********************************/
 
-include(STRIPE_BASE_DIR . '/includes/charge.php');
+include(SSP_BASE_DIR . '/includes/charge.php');
 
-function add_ajax_actions() {
-    add_action( 'wp_ajax_bsf_create_payment', 'bsf_create_payment_callback' ); 
-	add_action( 'wp_ajax_nopriv_bsf_create_payment', 'bsf_create_payment_callback' );
+function ssp_add_ajax_actions() {
+    add_action( 'wp_ajax_ssp_create_payment', 'ssp_create_payment_callback' ); 
+	add_action( 'wp_ajax_nopriv_ssp_create_payment', 'ssp_create_payment_callback' );
 }
 
-add_action( 'init', 'add_ajax_actions' );
+add_action( 'init', 'ssp_add_ajax_actions' );
```

Comment:
No need for a separate function `ssp_add_ajax_actions()` to register ajax actions